rllib/core/learner/learner.py

```diff
@@ -373,7 +373,7 @@ def _convert_batch_type(self, batch: MultiAgentBatch) -> NestedDict[TensorType]:
     @OverrideToImplementCustomLogic_CallToSuperRecommended
     def compile_results(
```

Comment:
Unrelated, but I think compile_results is too generic. It's almost like "produce_results" or "compile_return_values".
We return module_learner_stats in the end, so that is what we actually assemble here.